### DISCUSSION TITLE
Garmin Varia RCT716 support

### DISCUSSION THREAD

**Discussion by lorsch525** [maca88/SmartBikeLights#44] [Ideas]
Hey, could you please add in support for the Varia RCT716? I saw that the RTL 511 and RTL516 devices are supported, but not the newer one with the camera.

**Comment by maca88** [comment]
Hi, could you post what are the supported light modes for RCT716? If you have a touchscreen Edge, you can find out by installing Smart Bike Lights and set it on a 1 Field Layout. You should see buttons with numbers like this:
https://forums.garmin.com/cfs-file/__key/communityserver-discussions-components-files/10/Panel.PNG

I am assuming that it only support light mode "4", but I would like to be sure before adding it to the list of supported lights.

**Reply by lorsch525** [reply]
Unfortunately I only have an edge 530. It shows me a question mark on the data field when the light is on, it disappears when it is off
The Varia RCT716 only has that solid light mode, no blinking, no different brightness levels.

If that mode was "4", on the older RTL516 and RTL511, it should be the same, I think.

Couldn't it work otherwise if I simply make a configuration file for the RTL516 and use that with my RCT716?

**Comment by maca88** [comment]
Yes, try to select RTL516 or RTL511. If you don't get `Error: 3` then it means that it has the same mode. Please confirm in case it works so that I can add it to the list.

**Comment by DohbiK** [comment]
I can confirm that it works with the config for a Varia516

**Reply by maca88** [reply]
Thanks for letting me know. I've added Varia RCT716 to the configurator.
